hat/assets/js/apps/Iaso/domains/forms/components/LinkToForm.tsx

```diff
@@ -1,19 +1,33 @@
 import React, { FunctionComponent } from 'react';
 
 import { Link } from 'react-router';
+import { makeStyles } from '@material-ui/core';
 import { userHasPermission } from '../../users/utils';
 import { baseUrls } from '../../../constants/urls';
 import { useCurrentUser } from '../../../utils/usersUtils';
 
 type Props = {
-    formId: string;
+    formId: string | number;
     formName: unknown;
 };
+
+const useStyles = makeStyles(theme => ({
+    link: {
+        color: theme.palette.primary.main,
+        textDecoration: 'none',
+        '&:hover': { textDecoration: 'underline' },
+    },
+}));
 export const LinkToForm: FunctionComponent<Props> = ({ formId, formName }) => {
+    const classes: Record<string, string> = useStyles();
     const user = useCurrentUser();
     if (userHasPermission('iaso_forms', user)) {
         const formUrl = `/${baseUrls.formDetail}/formId/${formId}`;
-        return <Link to={formUrl}>{formName}</Link>;
+        return (
+            <Link className={classes.link} to={formUrl}>
+                {formName}
+            </Link>
+        );
     }
     return <>{formName}</>;
 };
```

Comment:
This should probably be made into a generic component for all links that appear in tables (at least), eg: users (emails) and polio grouped campaigns

Reply:
This is a component like `LinkToOrgUnit`, how do you want to make it more generic? it's in form domain